artemis-prometheus-metrics-plugin-servlet/pom.xml

```diff
@@ -35,19 +35,14 @@ under the License.
       <dependency>
          <groupId>org.apache.tomcat</groupId>
          <artifactId>tomcat-servlet-api</artifactId>
-         <version>8.5.5</version>
```

Comment:
Could you move the maven impovement to a separate commit?

Reply:
Separate PR created: https://github.com/rh-messaging/artemis-prometheus-metrics-plugin/pull/15